Examples/RTTI/RTTI_component/Examples/CppDynamic/RTTI_example.cpp

```diff
@@ -0,0 +1,50 @@
+/*++
+
+Copyright (C) 2020 ADSK
+
+All rights reserved.
+
+This file has been generated by the Automatic Component Toolkit (ACT) version 1.7.0-develop.
+
+Abstract: This is an autogenerated C++ application that demonstrates the
+ usage of the Dynamic C++ bindings of RTTI
+
+Interface version: 1.0.0
+
+*/
+
+#include <iostream>
+#include "rtti_dynamic.hpp"
+
+
+int main()
+{
+	try
+	{
+		std::string libpath = (""); // TODO: put the location of the RTTI-library file here.
+		auto wrapper = RTTI::CWrapper::loadLibrary(libpath + "/rtti."); // TODO: add correct suffix of the library
+		RTTI_uint32 nMajor, nMinor, nMicro;
+		wrapper->GetVersion(nMajor, nMinor, nMicro);
+		std::cout << "RTTI.Version = " << nMajor << "." << nMinor << "." << nMicro;
+		std::cout << std::endl;
+
+
+		auto zoo = wrapper->CreateZoo();
+		auto iter = zoo->Iterator();
+
+		using namespace RTTI;
+		while (auto animal = iter->GetNextAnimal()) {
+			if (auto tiger = rtti_cast<CTiger>(animal)) {
```

Comment:
@alexanderoster @martinweismann how do you feel about the name of this method (`<namespace>_cast`), I think it would be better to change this to something like `obj_cast` because it already resides in the namespace.

For example, using it on a component with namespace `Widgets` would currently look like this:

```c++
bool CheckWidget(Widgets::PBase pObj) {
    auto foo = Widgets::widgets_cast<Widgets::PFoo>(pObj);
    return foo ? foo->IsOk() : false;
}
```

Depending on the namespace, the method name can become awkward, especially when the namespace isn't imported.

Reply:
Good point.
Considering that class names in the ACT-C++ bindings are usually referred to as Namespace::CClassName, i think using `Namespace::obj_cast` is more consistent than `Namespace::Namespace_cast`.